lib/run.tsx

```diff
@@ -40,6 +40,7 @@ const makeUseMDXComponents = (more: RunOpts['components']): (() => ComponentOpts
     'code-tabs': Components.CodeTabs,
     'html-block': Components.HTMLBlock,
     embed: Components.Embed,
+    image: Components.Image,
```

Comment:
this is weird, idk what happened but i haven't peered into the MDX black box much yet